### ISSUE TITLE
Update single stack in 2.x

### ISSUE THREAD

**Issue by jinwg** [cloudreach/sceptre#678]
i understand  the update command is to allow user update single stack by resolving the dependent stack output without updating them as 1.x does, it is pretty useful feature when debug an specific stack issue.   in 2.x , sceptre seems try to update all dependent stacks but failed when no changes from its dependent stacks,  also tried  --ignore-dependencies but got the stopiteration exception.   if update try to updates all dependent stacks, then seem launch command already dose that.  that sound a bug of update command?  

```
sceptre update dev/shared-cluster.yaml
An error occurred (ValidationError) when calling the UpdateStack operation: No updates are to be performed.

sceptre --ignore-dependencies launch reuters-main/us-east-1/dev/shared-cluster.yaml
  File "/anaconda3/envs/py3.6/lib/python3.6/site-packages/sceptre/resolvers/stack_output.py", line 127, in resolve
    stack for stack in self.stack.dependencies if stack.name == friendly_stack_name
StopIteration
```

**Comment by henrist** [comment]
Partly overlapping with #533

**Comment by ngfgrant** [comment]
Thanks for the cross reference @henrist. I agree this is partially related.

**Comment by ngfgrant** [comment]
This will be resolved in 2.1.4
